srtcore/core.h

```diff
@@ -1204,6 +1204,7 @@ class CUDT
     sockaddr_any m_PeerAddr;   // peer address
     sockaddr_any m_SourceAddr; // override UDP source address with this one when sending
     uint32_t m_piSelfIP[4];    // local UDP IP address
+    int m_TransferIPVersion;   // AF_INET/6 that should be used to determine common payload size
```

Comment:
```suggestion
    int m_iTransferIPVersion;   // AF_INET/6 that should be used to determine the common payload size
```

Reply:
Actually not. The `i` marker defines an integer - counting - variable, while here it is used as a value that should be assigned to AF_INET or AF_INET6 symbol. If this was a decent C++ API, it would be an enum.